Gemfile

```diff
@@ -3,3 +3,5 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'unicode_utils' if (Gem::Version.new RUBY_VERSION) < (Gem::Version.new '2.4.0')
+
+gem 'opal', :git => 'https://github.com/opal/opal.git', :ref => 'tags/v1.0.0'
```

Comment:
I think we should use `d136ea8` to avoid issues: https://github.com/asciidoctor/asciidoctor-reveal.js/blob/acb2247380fe70e51dadcbb71568e2902d8865ce/Gemfile#L6

I'm very surprised that an extension transpiled with Opal 1.0.0 is working with Asciidoctor.js 2.x which is transpiled using Opal 0.11.